godump.go

```diff
@@ -43,22 +43,19 @@ var defaultDumper = NewDumper()
 var exitFunc = os.Exit
 
 var (
-	enableColor  = detectColor()
 	nextRefID    = 1
 	referenceMap = map[uintptr]int{}
 )
 
 // Colorizer is a function type that takes a color code and a string, returning the colorized string.
 type Colorizer func(code, str string) string
 
-// colorize is the default colorizer function.
-var colorize Colorizer = ansiColorize // default
+func formatNoColor(code, str string) string {
+	return str // No colorization
+}
 
-// ansiColorize colorizes the string using ANSI escape codes.
-func ansiColorize(code, str string) string {
-	if !enableColor {
-		return str
-	}
+// formatANSIColor colorizes the string using ANSI escape codes.
+func formatANSIColor(code, str string) string {
```

Comment:
This adds Color as a qualifier to "format" as well when it could just be "colorizeANSI"